processing/src/main/java/org/apache/druid/frame/read/columnar/ComplexFrameColumnReader.java

```diff
@@ -47,6 +51,38 @@ public class ComplexFrameColumnReader implements FrameColumnReader
     this.columnNumber = columnNumber;
   }
 
+  @Override
+  public Column readRACColumn(Frame frame)
```

Comment:
Lot of copypasta here, could it be factored out into a method that makes a `ComplexFrameColumn`, and then the RACColumn and ColumnPlus readers could call that?